v1alpha1/proto/grafeas.proto

```diff
@@ -1365,3 +1410,35 @@ service Grafeas {
     };
   }
 }
+
+service GrafeasProjects {
+  // Creates a new `Project`.
+  rpc CreateProject(CreateProjectRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      post:  "/v1alpha1/projects/{project_id=*}"
```

Comment:
Can you change this to be /v1alpha1/{name=projects/*} to be consistent with the other api definitions? here and below.

Reply:
Yeah. Was going back and forth on this. You are probably right though. Will fix this tomorrow.

Reply:
Fixed in 715d9e0